traitsui/testing/tester/ui_tester.py

```diff
@@ -10,15 +10,11 @@
 #
 import contextlib
 
-from traitsui.ui import UI
 from traitsui.testing._gui import process_cascade_events
 from traitsui.testing._exception_handling import reraise_exceptions
-from traitsui.testing.tester._ui_tester_register.default_registry import (
-    get_default_registry
-)
-from traitsui.testing.tester.registry import TargetRegistry
-from traitsui.testing.tester._ui_tester_register._traitsui_ui import (
-    register_traitsui_ui_solvers,
+from traitsui.testing.tester._ui_tester_registry.default_registry import (
+    get_default_registry,
+    _get_ui_registry
```

Comment:
I was thinking of adding these lines to `get_default_registry`
```
    register_traitsui_ui_solvers(
        registry=registry,
        target_class=UI,
        traitsui_ui_getter=lambda target: target,
    )
```
and then remove `_get_ui_registry` altogether.

Then `ui_tester` would not need to import `_get_ui_registry`, otherwise `_get_ui_registry` will need to be renamed to `get_ui_registry` to indicate that it is used outside of `traitsui.testing.tester._ui_tester_registry.default_registry`.